### PR TITLE
Alriune kill effect curtains

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

Adds the "curtains" that appear for a moment when someone dies to Alriune. Sprites by Lammar#4878

## Why It's Good For The Game

More fancy effects.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/alriune.dm [hunk 1]
```diff
@@ -78,7 +78,9 @@
 				if(ishuman(L))
 					var/mob/living/carbon/human/H = L
 					if(H.sanity_lost)
-						H.add_overlay(icon('ModularTegustation/Teguicons/tegu_effects.dmi', "alriune_kill"))
+						new /obj/effect/temp_visual/alriune_curtain(get_turf(H))
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/waw/alriune.dm:81]: Doesn't Alriune does this effect regardless of kill, it being her "you were hit by my attack" indicator?

Comment (dsmith328) [reply code/modules/mob/living/simple_animal/abnormality/waw/alriune.dm:81]: Also does this fail to check for the target being dead, resulting in potentially infinite overlays stacking on a dead insane person?

Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/abnormality/waw/alriune.dm:81]: The death check is above;
The curtains only appear on killed: For normal damage there's generic effect.

**Review by dsmith328 — APPROVED** [review]
Looks cool, sounds good 👍

**Pull request merged**